osutil/mountinfo_test.go

```diff
@@ -35,9 +35,12 @@ var _ = Suite(&mountinfoSuite{})
 
 // Check that parsing the example from kernel documentation works correctly.
 func (s *mountinfoSuite) TestParseMountInfoEntry1(c *C) {
-	entry, err := osutil.ParseMountInfoEntry(
-		"36 35 98:0 /mnt1 /mnt2 rw,noatime master:1 - ext3 /dev/root rw,errors=continue")
+	real := "36 35 98:0 /mnt1 /mnt2 rw,noatime master:1 - ext3 /dev/root rw,errors=continue"
+	canonical := "36 35 98:0 /mnt1 /mnt2 noatime,rw master:1 - ext3 /dev/root errors=continue,rw"
```

Comment:
Can you add a test case with escaped paths?

Reply:
Done